tests/io/test_cached.py

```diff
@@ -23,12 +23,16 @@ def default_cached_ds(tmpdir_factory, default_raw, lt_ctx):
     yield ds
 
 
-# A running Dask client can introduce a timing issue
-# between automatic closing of a numpy.memmap object and renaming
-# the underlying file
-def test_start_client():
+@pytest.fixture(scope='module', autouse=True)
+def client_in_background():
+    # A running Dask client can introduce a timing issue
+    # between automatic closing of a numpy.memmap object and renaming
+    # the underlying file
     with dd.LocalCluster() as cluster:
         client = dd.Client(cluster, set_as_default=False)
+        yield
+        # to fix "distributed.client - ERROR - Failed to reconnect to scheduler after 10.00 seconds, closing client"  # NOQA
+        client.close()
```

Comment:
So that means we always have a second client in the background (`autouse=True`)? How much time does that consume per module?

Reply:
As the fixture is defined in a single module, it should only be used there, right? I can double check if that is true.

Reply:
Ok, should be easy enough by trying to use it in a test in another module, right? Good point that it is not in conftest.py, so probably you are right. :-)

Reply:
> Ok, should be easy enough by trying to use it in a test in another module, right?

Yeah - I just tried by adding a debug print, and running another test after `test_cached` doesn't start the cluster.